iothub/service/src/Registry/Models/Device.cs

```diff
@@ -49,8 +51,8 @@ public Device(string id)
         /// <summary>
         /// Device's ETag.
         /// </summary>
-        [JsonProperty(PropertyName = "etag")]
-        public string ETag { get; set; }
+        [JsonProperty(PropertyName = "etag"), JsonConverter(typeof(NewtonsoftJsonETagConverter))] // NewtonsoftJsonETagConverter is used here because otherwise the ETag isn't serialized properly
```

Comment:
Can we make them separate attributes?
```suggestion
        [JsonProperty(PropertyName = "etag")]
        [JsonConverter(typeof(NewtonsoftJsonETagConverter))] // NewtonsoftJsonETagConverter is used here because otherwise the ETag isn't serialized properly
```

Reply:
Sure, I'll do that here and elsewhere